sdk/tools/log2lines/image.c

```diff
@@ -26,16 +26,18 @@ find_rossym_section(PIMAGE_FILE_HEADER PEFileHeader, PIMAGE_SECTION_HEADER PESec
     return NULL;
 }
 
+/* Unused.
 size_t
 fixup_offset(size_t ImageBase, size_t offset)
 {
     if (offset > ABS_TRESHOLD)
         offset -= ImageBase;
     return offset;
 }
+*/
 
 PROSSYM_ENTRY
-find_offset(void *data, size_t offset)
+find_offset(void *data, unsigned int offset)
```

Comment:
No, just change the local variable that's causing trouble, not all function signatures.

Reply:
Ah, I misunderstood the scope you meant.
Reverted.